packages/node_modules/@ciscospark/common/test/unit/spec/one-flight.js

```diff
@@ -5,6 +5,43 @@ import {assert} from '@ciscospark/test-helper-chai';
 
 describe(`common`, () => {
   describe(`@oneFlight`, () => {
+    it(`returns new promise on function with undefined keyFactory`, () => {
```

Comment:
The scenario which was causing the issue is that -> 
OneFlight was called on funcC1() with keyFactory as undefined. The promise is not resolved yet.
Now when you call funcC2() with keyFactory as undefined then it would assume that there is already a oneFlight i.e from above function call and would return that promise instead of actually calling funcC2(). I think that is a wrong behavior since it should allow calling funcC2(). Remember that both funcC1() and fncC2() are on the same object.

The fix:
1) My suggestion is that the keyFactory should never be called if undefined, as the weakmaps do not support undefined keys. If it does then it tends to lock the entire object and not that specific function.
2) If the keyFactory is undefined allow calling the function.